client/app/domain/zone/default-ttl/domain-zone-default-ttl.html

```diff
@@ -1,40 +1,33 @@
-            </div>
-
-            <div data-ng-if="!ctrl.loading">
-                <p data-ng-bind-html="tr('domain_tab_ZONE_default_ttl_heading', '<strong>' + ctrl.domain.displayName + '</strong>')"></p>
+    <oui-modal data-title="{{tr('domain_tab_ZONE_default_ttl_button')}}"
+               primary-action="ctrl.updateDefaultTTL()"
+               primary-label="{{tr('wizard_confirm')}}"
+               secondary-action="resetAction()"
+               secondary-label="{{tr('wizard_cancel')}}"
+               on-dismiss="resetAction()">
+        <div class="text-center" data-ng-if="ctrl.loading">
+            <oui-spinner></oui-spinner>
+        </div>
+        <div data-ng-if="!ctrl.loading">
+            <p data-ng-bind-html="tr('domain_tab_ZONE_default_ttl_heading', '<strong>' + ctrl.domain.displayName + '</strong>')"></p>
 
-                <form class="form-horizontal mt-4" name="ctrl.updateDefaultTtlForm">
-                    <div class="form-group" data-ng-class="{ 'has-error': ctrl.updateDefaultTtlForm.ttl.$dirty && ctrl.updateDefaultTtlForm.ttl.$invalid }">
-                        <label class="control-label col-md-3" for="nb-ttl"
-                               data-i18n-static="domain_tab_ZONE_default_ttl_default_value"></label>
-                        <div class="col-md-6">
-                            <input type="number" class="form-control" id="nb-ttl" name="ttl" required
-                                   data-ng-min="{{ctrl.autorizedValues.min}}" min="{{ctrl.autorizedValues.min}}"
-                                   data-ng-max="{{ctrl.autorizedValues.max}}" max="{{ctrl.autorizedValues.max}}"
-                                   data-ng-model="ctrl.zoneSOA.ttl"
-                                   data-ng-pattern="/^\d+$/">
-                            <small class="help-block"
-                                   data-ng-bind-html="tr('domain_tab_ZONE_default_ttl_default_value_conditions', [ctrl.autorizedValues.min, ctrl.autorizedValues.max])"
-                                   data-ng-if="ctrl.updateDefaultTtlForm.ttl.$dirty && ctrl.updateDefaultTtlForm.ttl.$invalid"></small>
-                        </div>
-                    </div>
-                </form>
+            <form name="updateDefaultTtlForm">
+                <oui-field label="{{tr('domain_tab_ZONE_default_ttl_default_value')}}">
+                    <oui-numeric id="nb-ttl"
+                                 name="ttl"
+                                 model="ctrl.zoneSOA.ttl"
+                                 required
+                                 min="{{ctrl.autorizedValues.min}}"
+                                 max="{{ctrl.autorizedValues.max}}">
+                    </oui-numeric>
+                </oui-field>
+            </form>
 
-                <div class="alert alert-info mt-5" role="alert"
-                     data-i18n-static="domain_tab_ZONE_default_ttl_info">
-                </div>
+            <div class="alert alert-info mt-5"
```

Comment:
may be use oui-message there in order to show the way to use it !

Reply:
:+1: